skrub/_reporting/_data/templates/column-summaries.css

```diff
@@ -121,14 +110,9 @@
 
 .columns-in-sample-tab .card-header {
     position: relative;
-    background-color: var(--color-background-highlight-primary);
-    border-block-end: var(--border-s) solid var(--color-border-highlight);
```

Comment:
@jeromedockes : do you think that this is the faulty change, regarding the border?

Reply:
no I think it is [this one](https://github.com/skrub-data/skrub/pull/1259/files#diff-bf4fee1dc6f544b4435e8bc709320ea16fff8cd09fbbe5f5e25a88d072f9c81fL94)

if it's ok for you I'll restore the deleted rule it but making the vertical margin much smaller

Reply:
Yes, but much smaller :).

Thanks!